apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/FissClaimRdaSink.java

```diff
@@ -46,6 +46,15 @@ RdaChange<RdaFissClaim> transformMessageImpl(String apiVersion, FissClaimChange
     return change;
   }
 
+  @Override
+  int getInsertCount(RdaFissClaim claim) {
+    return 1
```

Comment:
Maybe its obvious, but could we add a comment here that the 1 is for the parent claim row